docs/tutorials/tutorial-sql-null.md

```diff
@@ -0,0 +1,193 @@
+---
+id: tutorial-sql-null
+title: Null handling tutorial
+sidebar_label: Handling null values
+description: Introduction to null handling in Druid
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+This tutorial introduces the basic concepts of null handling for string and numeric columns in Apache Druid.
+The tutorial focuses on filters using the logical NOT operation on columns with NULL values.
+
+## Prerequisites
+
+Before starting this tutorial, download and run Apache Druid on your local machine as described in
+the [Local quickstart](index.md).
+
+The tutorial assumes you are familiar with using the [Query view](./tutorial-sql-query-view.md) to ingest and query data.
```

Comment:
i wonder if we need to call out that this tutorial expects `druid.generic.useDefaultValueForNull` and `druid.generic.useThreeValueLogicForNativeFilters` to be set to their default values of `false` and `true` respectively or if that would be confusing

Reply:
added a comment without detail since we don't want folks changing it if they're just getting started.